### ISSUE TITLE
Terminology: unify names for ring unit 

### ISSUE THREAD

**Issue by avekens** [metamath/set.mm#4512]
See also discussion in PR #4509:

We have several different names for `1r` already at the beginning of section "Ring unit":

* ring unit (in comment for `cur`) - used 16 times in set.mm
* multiplicative neutral element of a ring (in comment for ~df-ur) - used 8 times in set.mm
* unity element of a ring (in comment for ~ringidval) - used 3 times in set.mm ("unity" only ist used 23 times)
* multiplicative identity (of a ring) (in comment of ~dfur2) - used 18 times in set.mm
* ring unity (in comment of ~pridln1) - used 2 times in set.mm

We should choose one of them as the name we will use primarily, and should mention the others in the comment of the definition.

In Wikipedia (see https://en.wikipedia.org/wiki/Identity_element ), "the identity with respect to multiplication is called a **multiplicative identity** (often denoted as 1)".
But also "The distinction between additive and multiplicative identity is used most often for sets that support both binary operations, such as rings, integral domains, and fields. The multiplicative identity is often called **unity** in the latter context (a ring with unity)." 
A "unit", in contrast, is "an invertible element for the multiplication of the ring", see also definition of a "ring unit" `Unit` ( ~df-unit) in section "Divisibility", which is already an inconsistency within set.mm.

Therefore, I would propose to call it primarily "multiplicative identity" and alternatively "unity" or "unity element".

Consequently, the labes cur, df-ur and dfur2 should be changed, too. Maybe in 
* cmid, df-mid, dfmid2 with "mid" for "multiplictive identity" (df-mi is already used!) 
* or cue, df-ue, dfue2 with "ue" for "unity element"
* or c1r, df-1r, df1r2 analogous to the labels for `0g` (I would prefer this variant)

**Comment by tirix** [comment]
I agree with the direction!
The `1r` based labels would also be my preferred option.

**Comment by jkingdon** [comment]
Thanks for digging into this. I also would have said "multiplicative identity" so glad to hear that seems to be borne out by the sources you consulted.

As for "unit", the situation is worse than I thought, as some of the existing uses seem to be misapplying the word as you note.

**Comment by avekens** [comment]
I had a look at the literature, and saw that there is no uniform or even preferred terminology:
* N.Bourbaki and Marshall Hall call the multiplicative identity "unit" or "unit element" 
* Steven Roman and Beauregard & Fraleigh call an invertible element a "unit"
* Serge Lang calls the multiplicative identity "unit element" and an invertible element a "unit"

In Wikipedia [Unit (ring theory)](https://en.wikipedia.org/wiki/Unit_(ring_theory)): 

> In [algebra](https://en.wikipedia.org/wiki/Algebra), a **unit** or **invertible element** of a [ring](https://en.wikipedia.org/wiki/Ring_(mathematics)) is an [invertible element](https://en.wikipedia.org/wiki/Invertible_element) for the multiplication of the ring. That is, an element u of a ring R is a unit if there exists v in R such that v u = u v = 1 , where 1 is the [multiplicative identity](https://en.wikipedia.org/wiki/Multiplicative_identity);

and 

> Less commonly, the term unit is sometimes used to refer to the element 1 of the ring, in expressions like ring with a unit or unit ring, and also [unit matrix](https://en.wikipedia.org/wiki/Unit_matrix). Because of this ambiguity, 1 is more commonly called the "unity" or the "identity" of the ring, and the phrases "ring with unity" or a "ring with identity" may be used to emphasize that one is considering a ring instead of a [rng](https://en.wikipedia.org/wiki/Rng_(algebra)).

To avoid the ambiguity of the term "unit", I still propose to change the naming in (i)set.mm as discussed above.

**Comment by avekens** [comment]
> 
> Consequently, the labes cur, df-ur and dfur2 should be changed, too. Maybe in
> 
>     * cmid, df-mid, dfmid2 with "mid" for "multiplictive identity" (df-mi is already used!)
>     * or cue, df-ue, dfue2 with "ue" for "unity element"
>     * or c1r, df-1r, df1r2 analogous to the labels for `0g` (I would prefer this variant)

Unfortunately, c1r and df-1r are used already (for signed real constant 1). And "ur" still stands for "**U**nity element in a **R**ing"). Therefore, I will not change the label...

**Comment by jkingdon** [comment]
Is this issue ready to close now that #4584 is merged, or is there more to be done?

**Comment by avekens** [comment]
> Is this issue ready to close now that [#4584](https://github.com/metamath/set.mm/pull/4584) is merged, or is there more to be done?

No, I still want to align iset.mm with the revised set.mm. I will do it soon.

**Comment by jkingdon** [comment]
> > Is this issue ready to close now that [#4584](https://github.com/metamath/set.mm/pull/4584) is merged, or is there more to be done?
> 
> No, I still want to align iset.mm with the revised set.mm. I will do it soon.

Ah of course. I forgot that part hasn't happened yet.

**Comment by avekens** [comment]
Issue is done with PR #4601
